File: q03/03.js
var N = 100;
var cards = new Array(100)
for (var i = 0; i < N; i++) {
  cards[i] = false
}

for (var i = 2; i <= N; i++) {
  var j = i - 1;
  while (j < cards.length) {
    cards[j] = !cards[j];
    j += i
  }
}

for (var i = 0; i < N; i++) {
  if (!cards[i]) {
    console.log(i + 1);
  }
}

File: q03/03.swift
#!/usr/bin/swift

let N = 100
var cards = Array(repeating: false, count: N)

for i in 2...N {
    var j = i - 1
    while j < cards.count {
        cards[j] = !cards[j]
        j += i
    }
}

for i in 0..<N {
    if !cards[i] {
        print(i + 1)
    }
}
